### ISSUE TITLE
The "--no-use-reorder" option does not exist

### ISSUE THREAD

**Issue by slavafomin** [umpirsky/Symfony-Upgrade-Fixer#35]
Hello!

Thank you for this great library!

However, it tells me that `--no-use-reorder` option does not exists:

```
$ ./symfony-upgrade-fixer.phar fix ./ --no-use-reorder
                                                 
  [RuntimeException]                             
  The "--no-use-reorder" option does not exist.  

fix [--dry-run] [--] <path>
```

Why is that?

Thanks!

**Comment by umpirsky** [comment]
What version are you using, this is only in latest master I think.

Looks like we need to tag a new release. :)

**Comment by cuongthamhl** [comment]
This issue still exist. I downloaded https://github.com/umpirsky/Symfony-Upgrade-Fixer/releases/download/v0.1.5/symfony-upgrade-fixer.phar, but `php symfony-upgrade-fixer.phar --version` shows 0.1.4.

**Comment by umpirsky** [comment]
Just released [v0.1.6](https://github.com/umpirsky/Symfony-Upgrade-Fixer/releases/tag/v0.1.6). Thanks.
